Assignment2/ThomasVanDommelen.ttl

```diff
@@ -0,0 +1,14 @@
+@base <http://example.org/>.
+@prefix : <http://example.org/#>.
+
+:Class01  :includes :Sensor029;
+          :includes :Computer101.
+          
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Computer101 :hasOwner :User10A.
+
+:Measurement8401  :hasTemperature :29;
```

Comment:
There are syntactic errors here in the representation of literals. And be careful as well and provide datatypes for them. Check what other students have done.

Reply:
Please check what other students have done.